roles/rciam-metrics/templates/fastapi/fastapi.service.j2

```diff
@@ -8,7 +8,8 @@ Group=www-data
 
 WorkingDirectory={{ metrics_path }}
 Environment="PATH={{ metrics_path }}/.venv/bin"
-ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app
+ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix metrics_server_fastapi --bind {{ metrics_hostname }} --log-level {{ fastapi_log_level }}
```

Comment:
This depends on the previous comment. E.g. if we go with the `metrics_api_logging` approach:

`ExecStart={{ metrics_path }}/.venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker app.main:app --log-syslog --log-syslog-prefix {{ metrics_api_logging.syslog_identifier }} --bind {{ metrics_hostname }} --log-level {{ {{ metrics_api_logging.level }} }}`